README.md

```diff
@@ -1,5 +1,22 @@
-# crowdin-gateway
-Gateway for integrating with Crowdin for translation of content
+# Crowdin Gateway
 
+# Usage
+Gateway for integrating with Crowdin for translation of content.
 
+* The `content` of the request is assumed to be valid html.
+* The `metaData` of the request is assumed to be valid json.
 
+To interact with the API, you need valid security credentials; see [Access Tokens usage](https://github.com/NDLANO/auth/blob/master/README.md).
+
+For a more detailed documentation of the API, please refer to the [API documentation](https://staging.api.ndla.no).
```

Comment:
Om vi lenker til api.ndla.no for mer detaljert api dokumentasjon bare for å ikke finne det er jo det også ganske dumt. Hva med å lenke til staging, slik som det er gjort nå, og i stedet lage issues på å oppdatere readme filer når vi går i prod (slik at vi forhåpentligvis husker det).